src/framework/constraints.ml

```diff
@@ -78,7 +78,7 @@ struct
           | fd ->
             let c = S.context man fd d in
             sidel (FunctionEntry fd, c) d;
-            ignore (getl (Function fd, c))
+            demandl (Function fd, c)
```

Comment:
This raises an interesting point: Like this, demand is something that can only be triggered by the top-level in constraints and not by individual analyses: For this it would need to be exposed via a mechanism similar to `sideg`. But I guess this is not needed?

Other point: `spawn` sounds a lot like `create` and somewhat like `demand`. Are we happy with the nomenclature here?

Reply:
> This raises an interesting point: Like this, demand is something that can only be triggered by the top-level in constraints and not by individual analyses: For this it would need to be exposed via a mechanism similar to `sideg`. But I guess this is not needed?

As far as we're aware, we have no other uses for `demand` currently.
Since individual analyses cannot define unknowns with right-hand sides, they don't have anything useful to really `demand`.

> Other point: `spawn` sounds a lot like `create` and somewhat like `demand`. Are we happy with the nomenclature here?

`man.spawn` actually corresponds to the analyzed program spawning a new thread. I think that naming is still fine (and actually avoids confusing it with `demand`/`create` of the constraint systems).